lib/resources/orders.js

```diff
@@ -43,11 +43,11 @@ module.exports = function (api) {
     },
 
     create(params = {}, callback) {
-      let { amount, currency, receipt, payment_capture, notes,
+      let { amount, currency, receipt, payment_capture, notes, method,
             ...otherParams } = params
       currency = currency || 'INR'
 
-      if (!amount) {
+      if (!amount && method !== 'emandate') {
```

Comment:
We should really check for amount being exactly zero. This could match for amount being absent as well.